_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+header:
+  features: Features
+  showcase: Showcase
+  blog: Blog
+  community: Community
```

```suggestion
  community: Online-Community
```
Apparently the word community is more common with that prefix. That being said, it makes it even longer - I almost would advocate for something like "Netzwerk", "Vernetzen" or "Werde Teil der Community" because that feels more natural to me.

---

Vernetzen sounds good